lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -317,32 +317,53 @@ def increment_positions_on_all_items
         # Reorders intermediate items to support moving an item from old_position to new_position.
         def shuffle_positions_on_intermediate_items(old_position, new_position, avoid_id = nil)
           return if old_position == new_position
-          scope = acts_as_list_list
+          scope = acts_as_list_list.select(:id)
```

Comment:
Yes, pluck will be removed, it's redundant. Still, wouldnt it be performace-wise to only select ids for repositioning? Otherwise DB response will be a little bit larger and slower.